### PR TITLE
CIP-1694 | Move Acknowledgements into optional section

### PR DESCRIPTION
**Pull request by rphair**
We had a [request on Discord](https://discord.com/channels/971785110770831360/992011119872970762/1200547075285340322) (@klntsky) which was well received: to move "workshop participants" (and maybe sometimes lists of minor co-authors) to the ends of CIP documents as per the [optional sections](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure) legitimised in https://github.com/cardano-foundation/CIPs/pull/730.

This is also mandated by SEO principles like "source ordering" which is still an issue for properly indexing big documents with lots of names or code at the beginning.

Also CIP-0001 / CIP-9999 are updated to clear the way for similar Acknowledgements sections to be added in future CIPs / CPSs.  CIP-1694 is the one with the widest audience so let's clear this one first & then I/we can submit one more PR to update all others to follow suit: _after_ confirming this first change is well received by the community.  

@Ryun1 @Crypto2099 you've been managing some CIP relevant workshops so maybe we can also assemble a list of other documents to update in this PR discussion thread.  Another clue would be `grep`ing for the `<details>` tag to create the "accordion" sections.

@Hornan7 ~note this also updates the French version, but the commit hash is the same since the material isn't being re-translated.~ **edit, p.s.** French version has been removed from this PR since the English copy is changing a bit.

**p.s.** since this has long been associated with the Ledger Team perhaps @lehins @WhatisRT you would also confirm this particular change to CIP-1694 itself.

**p.p.s.** the "shortest common substring" algorithm used by `diff` doesn't always produce a straightforward result for moving one single large block of text elsewhere in a file.  _The **only** change_ submitted in https://github.com/cardano-foundation/CIPs/pull/763/commits/73c03910294b8e7f0aa128c1dac947367dc84121 is to move the entire `#### Acknowledgements` section from near the beginning (before the _Motivation_) to near the end (before the _Copyright_).

---

([updated CIP-1694 in English](https://github.com/rphair/CIPs/blob/cip-1694-move-workshop-participants/CIP-1694/README.md))
([updated CIP-0001](https://github.com/rphair/CIPs/blob/cip-1694-move-workshop-participants/CIP-0001/README.md))
([updated CIP-9999](https://github.com/rphair/CIPs/blob/cip-1694-move-workshop-participants/CIP-9999/README.md))

### CODE REVIEW

File: CIP-1694/README.md [hunk 1]
```diff
@@ -55,1950 +55,1950 @@ Voting rights will be based on the total Ada that is delegated, as a whole numbe
 
 The most crucial aspect of this proposal is therefore the notion of **"one Lovelace = one vote"**.
```
Comment (rphair) [inline CIP-1694/README.md:57]:
```suggestion

For the many contributors to this proposal, see [Acknowledgements](#acknowledgements).
```

**Review by rphair — COMMENTED** [review]
@lehins @WhatisRT @JaredCorduan - Editors in the CIP meeting today considered how important (politically and personally) it is for the community to be aware of these contributors, even in cases where nontechnical or hurried readers won't see the whole document.

We decided in these cases CIPs / CPSs can include a link to a latter Acknowledgements section "above the fold" by author's preference.  I believe this can be done in the _Abstract_ because community contribution can be considered part of a document's summary.

Please have a look at this suggestion and let us know if you think it's acceptable or if you would recommend different markup:

**Comment by WhatisRT** [comment]
Looks good, especially with the added reference to the Acknowledgements section.

File: CIP-1694/README.md (hunk 1 above)
Comment (rphair) [reply CIP-1694/README.md:57]: thanks @WhatisRT - committing as per https://github.com/cardano-foundation/CIPs/pull/763#issuecomment-1956388271.

**Comment by Hornan7** [comment]
I'll make sure the French version is updated within 3 days of the merge.

**Comment by rphair** [comment]
@Ryun1 @Crypto2099 adding `Last Check` for next meeting since this has author & community approval & I believe just needs to be double checked by the editors.

**Review by Ryun1 — APPROVED** [review]
I agree that this is a very sensible change, thanks for pushing forward with this @rphair 💪

**Review by Crypto2099 — APPROVED** [review]
These changes make sense and will make the CIPs much more machine-readable (i.e. search engine optimization) while formalizing co-author and contributor acknowledgements as part of the CIP/CPS process.

**Comment by rphair** [comment]
**epilogue**: pinging other devs on Discord (in [this post](https://discord.com/channels/971785110770831360/992011119872970762/1214634845939171398)) to see which other proposals we might want to do this with (cc @Ryun1 @Crypto2099 @klntsky).

**Pull request merged**